_pkgdown.yml

```diff
@@ -57,3 +57,9 @@ reference:
 - title: Package global state
   contents:
   - clear_cache
+
+authors:
+  Ramiro Magno:
+    href: https://www.pattern.institute/team/rmagno/
+  Pattern Institute:
+    href: https://www.pattern.institute
```

Comment:
@ramiromagno - We already have this information in the Description file. Do we need to repeat this information in the _yml file?

Reply:
Yes, otherwise the authors on the side navigation bar won't be hyperlinked.

Reply:
I don't think any of us are hyperlinked. If we want hyperlinks, we all need to add our information here. Other Pharmaverse packages don't have this either. Would you consider removing it now and adding it back after the release?

Reply:
I believe it should be at each author's discretion to display hyperlinks to their pages. I guess other authors are welcome to add their links if they wish. I think the question should be posed the other way around: is there any reason not to want links?